cmd/appgw-ingress/main.go

```diff
@@ -72,13 +64,22 @@ func main() {
 		glog.Fatal("Error parsing command line arguments:", err)
 	}
 
+	// Setup enviroment
```

Comment:
This comment technically should be - fetching environment variables etc.  But I even wonder if it is necessary. Seems the line below is fairly self-explanatory ;)